test/unit/SourceLocator/Type/AutoloadSourceLocatorTest.php

```diff
@@ -4,6 +4,7 @@
 
 namespace Roave\BetterReflectionTest\SourceLocator\Type;
 
+use phpDocumentor\Reflection\DocBlock\ExampleFinder;
```

Comment:
Let's please use a test asset to verify this: relying on the paths defined internally by phpDocumentor is not safe, as that's not part of the API.

Reply:
@moufmouf nah, let's leave composer completely alone and register an autoloader in the test:

Pseudo-code:

```php
spl_autoload_register(function (string $className) : bool {
    if ($className !== The\Test\Asset::class) { return false; }

    if (file_exists('path/to/non/existing/stuff.php')) {
        return false;
    }

    require_once 'path/to/existing/stuff.php';
    return true;
});
```

Reply:
Woot! Excellent idea! I'm on it.

Reply:
Remember: `class_exists($className, false)` MUST yield `false` after running this.